nucypher/characters/unlawful.py

```diff
@@ -61,15 +65,18 @@ def from_target_ursula(cls,
         crypto_power = CryptoPower(power_ups=target_ursula._default_crypto_powerups)
 
         if claim_signing_key:
-            crypto_power.consume_power_up(SigningPower(pubkey=target_ursula.stamp.as_umbral_pubkey()))
+            crypto_power.consume_power_up(SigningPower(public_key=target_ursula.stamp.as_umbral_pubkey()))
 
         if attach_transacting_key:
-            cls.attach_transacting_key(blockchain=target_ursula.blockchain)
+            cls.attach_transacting_key(blockchain=target_ursula.policy_agent.blockchain)
 
 
         vladimir = cls(is_me=True,
                        crypto_power=crypto_power,
                        db_filepath=cls.db_filepath,
+                       domains=[':TEMPORARY_DOMAIN:'],
```

Comment:
We should use the ``TEMPORARY_DOMAIN` constant in `nucypher/config/constants.py`. #2207 makes the value of the constant for this domain lowercase

Reply:
Good eye!

Reply:
Please bring these comments over to #1741.